docs/helm-chart-submission.md

```diff
@@ -0,0 +1,21 @@
+# Submission of Helm Charts for verification
+
+OpenShift Helm Charts repository hosts Helm Charts that are available by default with OpenShift. You can use this repository to submit the charts that need to be certified through a Pull Request. Once your Pull Request is merged, a CI/CD pipeline is created which publishes the chart in the GitHub releases which is further reflected on the Helm charts [repository index](http://charts.openshift.io/).
+
+The submission process of a Helm Chart for Red Hat Helm Repository and Certification has been documented on the [Red Hat Helm Repository](https://github.com/openshift-helm-charts/charts). Please note the instructions mentioned on the repository before submitting a chart.
+
+The follow options are available for submitting a Chart for inclusion in Red Hat Helm Repository and Certification: 
+
+| Option                                       | Description                                                                                                             |
+|----------------------------------------------|-------------------------------------------------------------------------------------------------------------------------|
+| Helm Chart Tarball or the extracted Tarball  | Submit your Chart with the specific or the extracted tarball. Here the chart-verifier's report is optional.             |
+| Chart Verification Report only               | Submit your Chart Verification Report without the Chart.                                                                |
+| Both Chart Verification Report and the Chart | Submit both the Chart Verification Report and the Chart by placing the source or tarball under the versioned directory. |
+
+> **_NOTE:_**  A verifier report is an integral part of the submission process. With the options that do not require a report, a report will be generated as part of the submission process.
+
+If the Pull Request fails, an exception process will be started if the report contains one or more failures or has missing tests. If the report can be submitted without a chart, the report should be run against the chart in the corresponding final location. The verifier will record the `chart-uri` specified during the report run and, in the absence of a submitted chart, the `chart-uri` would be used for the publication.
```

Comment:
To avoid future tense, I recommend rephrasing to "If the Pull Request fails, an exception process starts if the report contains one or more failures or has missing tests." 

Also, you have two conditional phrases here: if the PR fails, and if the report contains failures. In which case does the exception process starts? If both (and not one or the other), consider rephrasing to something along the lines of: 
"An exception process starts if the Pull request fails, if the report contains one or more failures, or has missing tests."

Reply:
Done!